CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* FE-378: fix fields containing spaces during index creation on web UI.
```

Comment:
This is only to trim spaces in the start and end?

Reply:
yeah, as we say enter comma separated field, we should trim that. I think spaces in field name itself could be valid on a db level